src/uwtools/config.py

```diff
@@ -752,3 +753,35 @@ def create_config_obj(
                     raise ValueError(err_msg)
                 # Dump to file:
                 dump_method(path=outfile, cfg=config_obj)
+
+
+def log_and_error(msg: str, log: Logger) -> None:
```

Comment:
Probably this should be marked private, as `_log_and_error()`, since we don't intend for it to be called from outside this module.

Reply:
Should the last three functions in `config.py` be re-ordered to have `# Private functions` and `# Public functions` with their respective functions sorted underneath them?

Reply:
I like that 👍